cmd/wallet-web/src/pages/Dashboard.vue

```diff
@@ -104,8 +124,9 @@ export default {
   },
   data() {
     return {
-      cards: [],
-      cjson: credentialDisplayData,
+      credentials: [],
+      processedCredentials: [],
+      supportedCredentials: credentialDisplayData,
```

Comment:
no need to save it as instance variable, you used it once and discard it.

Reply:
It does not work that way in Vue. Yet, I agree we don't need to put it in data, since it's static. We should refer to https://stackoverflow.com/questions/45565349/how-to-access-external-json-file-objects-in-vue-js-app, probably @talwinder50

Reply:
I have a todo for this, can do in a separate pr